Tasks/ServiceFabricDeploy/Strings/resources.resjson/en-US/resources.resjson

```diff
@@ -45,6 +46,20 @@
   "loc.input.label.MaxPercentUnhealthyDeployedApplications": "MaxPercentUnhealthyDeployedApplications",
   "loc.input.label.UpgradeTimeoutSec": "UpgradeTimeoutSec",
   "loc.input.label.ServiceTypeHealthPolicyMap": "ServiceTypeHealthPolicyMap",
+  "loc.input.label.configureDockerSettings": "Configure Docker settings",
```

Comment:
Yes, because there are a bunch of input properties that were added to the task.

---
In reply to: [136702516](https://github.com/Microsoft/vsts-tasks/pull/5076#discussion_r136702516) [](ancestors = 136702516)